libraries/Parsers/Microsoft.Bot.Builder.Parsers.LU/Parser/EntitySection.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Why are we allowing this? I think we should ahve the xml docs unless there is something I'm missing.